core/python3AiAction/pythonrunner.py

```diff
@@ -0,0 +1,100 @@
+"""Executable Python script for running Python actions.
+
+/*
```

Comment:
why did we need to clone and own this file?

Reply:
> why did we need to clone and own this file?

because the old Dockerfile added pythonrunner.py through releases 1.0.3, it could not apply this modification.